test-framework/common/src/main/java/io/quarkus/test/common/ResourceArg.java

```diff
@@ -0,0 +1,22 @@
+package io.quarkus.test.common;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Uses to define arguments of for {@code QuarkusTestResource}
+ *
+ * see {@link QuarkusTestResource#initArgs()}
+ */
+@Target(ElementType.METHOD)
```

Comment:
I think this should be declared as `@Target({})`.

Reply:
Ah yeah, that's definitely better